types/spectest/tests/consensusdata/proposer/versioned_block_spectest.go

```diff
@@ -60,6 +60,10 @@ func (test *ProposerSpecTest) Run(t *testing.T) {
 			require.NotNil(t, vBlk.Bellatrix)
 			blkSSZ, err = vBlk.Bellatrix.MarshalSSZ()
 			require.NoError(t, err)
+		case spec.DataVersionCapella:
```

Comment:
did we add any tests?

Reply:
@GalRogozinski I added pretty much the same question on top of one of your old comments, in case you missed - 
I know we're trying to keep small PRs but adding Capella but not adding Capella tests on the same PR kinda misses the point imho.

Reply:
I will write the same answer again.
This is not a merge to `main` branch.
When we merge the long-lived branch to `main` it will be completed.
We do have a happy-flow example though, mainly for comments so that the subsequent PRs will be created with initial feedback.